linker/Mono.Linker.Steps/MarkStep.cs

```diff
@@ -153,7 +153,11 @@ void ProcessQueue ()
 			while (!QueueIsEmpty ()) {
 				MethodDefinition method = (MethodDefinition) _methods.Dequeue ();
 				Annotations.Push (method);
-				ProcessMethod (method);
+				try {
+					ProcessMethod (method);
+				} catch (Exception e) {
+					throw new Exception (string.Format ("Error processing method: '{0}' in assembly: '{1}'", method.FullName, method.Module.Name), e);
```

Comment:
@marek-safar Do you know if there is an existing exception type that would make sense to use?  Or should I introduce a "FatalLinkerException" ?